Test/Unit/Helper/TestHelper.php

```diff
@@ -1,29 +0,0 @@
-<?php
-
-namespace Bolt\Boltpay\Test\Unit\Helper;
-
-use PHPUnit\Framework\TestCase;
-
-class TestHelper extends TestCase
```

Comment:
Was nothing using this old `TestHelper.php` before? I don't see any updated `use` statements.

Reply:
It was used from the same folder, in one one place only so far, SessionTest.php. I moved it one folder up, more logical place to be used in other tests / folders.